tests/fields/test_date_input_field.py

```diff
@@ -11,6 +11,17 @@
 from tbxforms.fields import DateInputField
 
 
+def test_compress_invalid_fields():
+    """Verify compress raises an error on an invalid date."""
+    day = 30
+    month = 2
+    year = 2021
+    field = DateInputField()
+    with pytest.raises(ValidationError) as err:
+        field.compress([day, month, year])
+    assert str(err) == "day is out of range for month"
```

Comment:
To get this passing you should be able to compare with `err.message` instead of `str(err)`